.travis.yml

```diff
@@ -23,8 +23,8 @@ script:
 
   - npm --silent run buildApps
 
-  - npm --silent run deploy-s3 -- -b cesium-dev -d cesium/$TRAVIS_BRANCH --confirm -c 'no-cache'
-  - npm --silent run deploy-status -- --status success --message Deployed
+  #- npm --silent run deploy-s3 -- -b cesium-dev -d cesium/$TRAVIS_BRANCH --confirm -c 'no-cache'
```

Comment:
Did you mean to include this change?

Reply:
@OmarShehata There was a reason for it at the time but I assume we can get rid of it at this point?

Reply:
This was just needed for the fork so CI would pass, it's fine to revert now.